src/main/java/io/jenkins/plugins/orka/client/OrkaClient.java

```diff
@@ -38,12 +40,21 @@
 
     private String endpoint;
     private String token;
+    private OkHttpClient client;
 
     public OrkaClient(String endpoint, String email, String password) throws IOException {
+        client = clientBase.newBuilder().readTimeout(defaultHttpClientTimeout, TimeUnit.SECONDS).build();
```

Comment:
You should be able to call the other ctor here with `defaultHttpClientTimeout` as argument.
As this is the only thing different.

Reply:
And you can use `this` for consistency. Although we do not have a style validation for that